sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -489,6 +489,38 @@ public void processElement(ProcessContext c) throws Exception {
   }
 
   @Test
+  public void testWriteEmptyPCollection() throws Exception {
```

Comment:
Was there also a bug about writing empty collections that is addressed in this PR?